dev/bots/check_code_samples.dart

```diff
@@ -319,7 +319,17 @@ final Set<String> _knownMissingTests = <String>{
   'examples/api/test/material/color_scheme/dynamic_content_color.0_test.dart',
   'examples/api/test/material/platform_menu_bar/platform_menu_bar.0_test.dart',
   'examples/api/test/material/flexible_space_bar/flexible_space_bar.0_test.dart',
-  'examples/api/test/material/selection_area/selection_area.0_test.dart',
+  'examples/api/test/material/chip/deletable_chip_attributes.on_deleted.0_test.dart',
+  'examples/api/test/material/expansion_panel/expansion_panel_list.expansion_panel_list_radio.0_test.dart',
+  'examples/api/test/material/search_anchor/search_anchor.0_test.dart',
+  'examples/api/test/material/search_anchor/search_anchor.1_test.dart',
+  'examples/api/test/material/search_anchor/search_anchor.2_test.dart',
+  'examples/api/test/material/scaffold/scaffold_messenger.of.0_test.dart',
+  'examples/api/test/material/scaffold/scaffold_messenger.0_test.dart',
+  'examples/api/test/material/scaffold/scaffold_state.show_bottom_sheet.0_test.dart',
+  'examples/api/test/material/scaffold/scaffold_messenger_state.show_material_banner.0_test.dart',
+  'examples/api/test/material/scaffold/scaffold_messenger.of.1_test.dart',
+  'examples/api/test/material/scaffold/scaffold_messenger_state.show_snack_bar.0_test.dart',
```

Comment:
Probably all those files should not be there anymore because these examples were added recently.
```suggestion
```